onnx/backend/test/case/node/rmsnormalization.py

```diff
@@ -0,0 +1,159 @@
+# Copyright (c) ONNX Project Contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+from __future__ import annotations
+
+import numpy as np
+
+import onnx
+from onnx.backend.test.case.base import Base
+from onnx.backend.test.case.node import expect
+
+
+# RMS normalization's reference implementation
+def _rms_normalization(X, W, axis=-1, epsilon=1e-5):  # type: ignore
+    shape = X.shape
+    rank = len(shape)
+    if axis < 0:
+        # If axis = -1 and rank of X is 4,
+        # the axis is changed to -1 + 4 = 3,
+        # which means the last axis.
+        axis = axis + rank
+
+    # Parameter used to convert N-D tensor RMS normalization to equivalent 2-D matirx operations.
+    row_number = np.prod(shape[:axis]).astype(np.int64)
+    col_number = np.prod(shape[axis:]).astype(np.int64)
+
+    # After reshaping input tensor X into a matrix,
+    # RMS normalization is equivalent to conducting
+    # standardization on each column vector (s.t. each
+    # column has zero mean and unit variance).
```

Comment:
I don't think this is true. It doesn't change the mean to zero. As for variance: I am not sure, without doing the math. It is better to describe what it does operationally: divide each column vector by its rms value (with the epsilon adjustment to avoid divide-by-zero).